content/en/docs/instrumentation/cpp/manual.md

```diff
@@ -2,10 +2,24 @@
 title: Manual Instrumentation
 linkTitle: Manual
 weight: 30
+description: Manual instrumentation for OpenTelemetry C++
 ---
 
-Manual instrumentation is the process of adding observability code to your
-application.
+[Manual instrumentation](https://opentelemetry.io/docs/concepts/instrumentation/manual/)
+is the process of adding observability code to your application yourself. If you
+are developing a standalone process or service, then you would take a dependency
+on the API and the SDK.
+
+If you are developing a library, that wants to export telemetry data using
+OpenTelemetry, it MUST only depend on the API and should never configure or
+depend on the OpenTelemetry SDK.
+
+This way libraries will obtain a real implementation of OpenTelemetry only if
+the user application is configured for it and otherwise will function as without
+added observability.
```

Comment:
Combine the paragraphs since they are both dealing with libraries?

```suggestion
depend on the OpenTelemetry SDK. In this way, libraries will obtain a real implementation of OpenTelemetry only if
the user application is configured for it and otherwise will function as without
added observability.
```

I can't quite make sense of this last sentence either.

Reply:
> In this way, libraries will obtain a real implementation of OpenTelemetry only if the user application is configured for it

This was the original sentence I copied from one of the language SIG pages. I think the message is that the library making only use of the API works just as if the API was never used until someone adds the SDK to the application. It's really hard to express that in a short sentence. But we can drop it, since it is explained in more detail in the library guides.